packages/grid/data-grid/src/tests/filtering.DataGrid.test.tsx

```diff
@@ -189,12 +189,51 @@ describe('<DataGrid /> - Filter', () => {
 
   describe('date operators', () => {
     [
-      { operator: 'is', value: new Date(2000, 11, 1), expected: ['12/1/2000'] },
-      { operator: 'not', value: new Date(2000, 11, 1), expected: ['1/1/2001', '1/1/2002'] },
-      { operator: 'after', value: new Date(2001, 0, 1), expected: ['1/1/2002'] },
-      { operator: 'onOrAfter', value: new Date(2001, 0, 1), expected: ['1/1/2001', '1/1/2002'] },
-      { operator: 'before', value: new Date(2001, 0, 1), expected: ['12/1/2000'] },
-      { operator: 'onOrBefore', value: new Date(2001, 0, 1), expected: ['12/1/2000', '1/1/2001'] },
+      { operator: 'is', value: '' },
+      { operator: 'is', value: undefined },
+      { operator: '', value: '2000-12-01' },
+      { operator: '', value: '' },
+    ].forEach(({ operator, value }) => {
+      it(`should not filter when operator='${operator}' and value='${value}'`, () => {
+        render(
+          <TestCase
+            value={value}
+            operator={operator}
```

Comment:
This was confusing when reviewing the PR, it would have been clearer with:

```suggestion
            operatorValue={operatorValue}
```

Reply:
Updated. IMHO, `operator` should be the name of the operator (is, after, before, contains) and `operatorValue` is the argument that feeds the operator.

Reply:
Yeah, it's confusing, it took me a while to figure this out. Not changing the terminology in the test is already one step forward toward clarity :)